examples/ads/ads.amp.html

```diff
@@ -0,0 +1,1937 @@
+<!doctype html>
```

Comment:
I don't think this file should have such a large diff. Have you pulled in the latest master? Or perhaps your text editor did some automatic formatting?

Oh! This file was incorrectly added. Seems like you have both `examples/ads/ads.amp.html` and `examples/ads.amp.html`. You only need `examples/ads.amp.html`. Please remove this file 😄 👍

Reply:
Hello @torch2424 ,

I have removed the file `examples/ads/ads.amp.html` from amp code. Please review the code again.